reference-ril/reference-ril.c

```diff
@@ -3064,20 +3064,25 @@ onRequest (int request, void *data, size_t datalen, RIL_Token t)
                it will call GET_CURRENT_CALLS and determine success that way */
             RIL_onRequestComplete(t, RIL_E_SUCCESS, NULL, 0);
             break;
-        case RIL_REQUEST_SWITCH_WAITING_OR_HOLDING_AND_ACTIVE:
+        case RIL_REQUEST_SWITCH_WAITING_OR_HOLDING_AND_ACTIVE: {
```

Comment:
Since we don't declare any new variable in this case block, I think we won't have to add `{` for this case block.